config/v1/types_apiserver.go

```diff
@@ -184,7 +184,8 @@ type APIServerEncryption struct {
 	Type EncryptionType `json:"type,omitempty"`
 }
 
-// +kubebuilder:validation:Enum="";identity;aescbc
```

Comment:
@deads2k I removed the kubebuilder validation because it felt redundant to have both validations in place, is that correct?